### PR TITLE
Integrated processing and other modules in docker-compose

### PR DESCRIPTION
**Pull request by samarthdd**
Integration of modules in docker-compose file

Modules include: 
cdr-f2f
jupyter notebook
elastic search
Kibana
Minio

Updated readme

### CODE REVIEW

File: cdr_plugin_folder_to_folder/common_settings/Config.py [hunk 1]
```diff
@@ -38,9 +38,9 @@ def load_values(self):
 
     def check_config(self):
         if folder_not_exists(self.hd1_location):
-            raise Exception("Folder not found: HD1")
+            raise Exception(f"{self.hd1_location} Folder not found: HD1")
         if folder_not_exists(self.hd2_location):
-            raise Exception("Folder not found: HD2")
+            raise Exception(f"{self.hd2_location}Folder not found: HD2")
```
Comment (ggrig) [inline cdr_plugin_folder_to_folder/common_settings/Config.py:43]: Please add a space between } and Folder

**Review by ggrig — CHANGES_REQUESTED** [review]
A minor issue noticed

File: cdr_plugin_folder_to_folder/common_settings/Config.py (hunk 1 above)
Comment (samarthdd) [reply cdr_plugin_folder_to_folder/common_settings/Config.py]: refactored

File: cdr_plugin_folder_to_folder/pre_processing/Pre_Processor.py [hunk 2]
```diff
@@ -129,9 +129,11 @@ def update_status(self):
             raise error
 
 # Fast API
+
 from fastapi import FastAPI
 app = FastAPI()
-@app.get("/process")
+
+@app.get("/pre-process")
```
Comment (DinisCruz) [inline cdr_plugin_folder_to_folder/pre_processing/Pre_Processor.py:136]: refactor this to a separate class, use the ```router = APIRouter()``` and add these methods to the Server.py

Comment (DinisCruz) [reply cdr_plugin_folder_to_folder/pre_processing/Pre_Processor.py:136]: we also need to create a Client API to easily invoke this from the Jupyter notebooks

**Review by DinisCruz — APPROVED** [review]

**Pull request merged**